api/scpca_portal/management/commands/create_portal_metadata.py

```diff
@@ -36,7 +33,16 @@ def create_portal_metadata(self, **kwargs):
             Project.objects.all(), common.GENERATED_PORTAL_METADATA_DOWNLOAD_CONFIG
         )
 
-        if kwargs["clean_up_output_data"]:
-            self.clean_up_output_data()
+        if computed_file:
+            logger.info("Saving the instance to the database")
+            computed_file.save()
+
+            if kwargs.get("update_s3", settings.UPDATE_S3_DATA):
```

Comment:
```suggestion
            if update_s3:
```
lets break up kwarg and add typehints to the function signature too.
```python
def create_portal_metadata(clean_up_output_data: bool, update_s3: bool, **kwargs):
```

Reply:
See at 609dc068afede2c66c6a427d0435792e7661a2fc.